shadow-dom/focus/text-selection-with-delegatesFocus.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-actions.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+
+<body>
+    <x-shadow id="withoutFocus"></x-shadow>
+    <x-shadow id="withFocus"></x-shadow>
+</body>
+
+<script>
+'use strict';
+
+/**
+ * build shadow-root with delegates focus, a focusable element, and an (ideally) selectable text
+ */
+function buildShadowRootWithSelectableText( element, shouldDelegateFocus ) {
+    element.attachShadow({ mode: 'open', delegatesFocus: shouldDelegateFocus });
+    const span = document.createElement('span');
+    span.textContent = 'Example Text to Select ';
+    const button = document.createElement('button');
+    button.textContent = 'Button';
+    element.shadowRoot.append(span);
+    element.shadowRoot.append(button);
+}
+
+/**
+ * command to select text in shadow-root
+ */
+function selectText(element, start, end) {
+    getSelection().empty();
+    const actions = new test_driver.Actions();
+    actions.pointerMove(start, 0, {origin: element});
+    actions.pointerDown();
+    actions.pointerMove(end, 0, {origin: element});
+    actions.pointerUp();
+    return actions.send();
+}
+
+promise_test(async () => {
+    const xShadow = document.getElementById('withoutFocus');
+    buildShadowRootWithSelectableText(xShadow, false);
+    await selectText(xShadow, -40, 45)
+    const s = getSelection();
+    assert_greater_than(s.toString().length, 0);
```

Comment:
I'm only confirming that some text is selected (and not a specific set of text), because it appears that the text that is selected can be different based on the browser (at least, with this text selection method, which admittedly I copied and modified from elsewhere in the code). It's also not super important that any specific text be selected, rather, just that _some_ text is selected. The tests pass and fail in the way as described in the summary.

Reply:
This seems ok to me as-written. There will be differences based e.g. on font size, but the important bit is just that some text gets selected.